public/content/it/index.yml

```diff
@@ -0,0 +1,4 @@
+title: Il tour di D
```

Comment:
I'm not a great fan of long titles, but you are right that the current one sounds quite bad, so I'll change it. Thanks.